src/Clock.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends React.Component<Props, State> {
+  timerId = 0;
+
+  state = {
+    date: new Date(),
+  };
```

Comment:
State should be first and then class fields and methods
```suggestion
   state = {
    date: new Date(),
   };
   
  timerId = 0;

  
```

Reply:
Done!